vector/src/main/java/im/vector/riotx/features/crypto/quads/SharedSecuredStorageKeyFragment.kt

```diff
@@ -48,8 +43,7 @@ class SharedSecuredStorageKeyFragment @Inject constructor(
         ssss_restore_with_key_text.text = getString(R.string.enter_secret_storage_input_key)
 
         ssss_key_enter_edittext.editorActionEvents()
-                .debounce(300, TimeUnit.MILLISECONDS)
-                .observeOn(AndroidSchedulers.mainThread())
```

Comment:
Also, why removing mainThread? the submit method is changing some ui, we might get into trouble if we are not switching back to the ui thread

Reply:
we are in the main thread, editorActionEvents() is supposed to emit only on main thread (no?) and debounce does not change the thread, so I think observeOn(AndroidSchedulers.mainThread()) is not necesarry. But maybe I'm wrong?

Reply:
Deboune and throttle does change thread by default

Reply:
Ok, I reverted this change